### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1552,18 +1552,26 @@ msgid ""
...
 "    git push <name>\n"
 msgstr ""
+"Немає налаштованого місця призначення надсилання.\n"
+"Або вкажіть URL-адресу з командного рядка, або налаштуйте віддалений репозиторій за допомогою\n"
+"\n"
+"    git remote add <ім'я> <адреса>\n"
+"\n"
+"а потім виконайте надсилання з використанням імені віддаленого репозиторія\n"
+"\n"
+"    git push <ім'я_віддаленого_репозиторія>\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Немає налаштованого місця призначення надсилання.\n"
"Або зазначте URL-адресу з командного рядка, або налаштуйте віддалений репозиторій за допомогою\n"
"\n"
"    git remote add <ім'я> <адреса>\n"
"\n"
"а потім виконайте надсилання, використовуючи імʼя віддаленого репозиторія\n"
"\n"
"    git push <ім'я>\n"
```

File: po/uk.po [hunk 2]
```diff
@@ -1552,18 +1552,26 @@ msgid ""
...
 
 #: builtin/push.c:678
 msgid "--all can't be combined with refspecs"
-msgstr ""
+msgstr "--all не можна комбінувати зі специфікатором посилання"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "--all не можна комбінувати зі специфікаторами посилань"
```

File: po/uk.po [hunk 3]
```diff
@@ -1552,18 +1552,26 @@ msgid ""
...
 
 #: builtin/push.c:684
 msgid "--mirror can't be combined with refspecs"
-msgstr ""
+msgstr "--mirror не можна комбінувати зі специфікатором посилання"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "--mirror не можна комбінувати зі специфікаторами посилань"
```

File: po/uk.po [hunk 4]
```diff
@@ -1552,18 +1552,26 @@ msgid ""
...
 
 #: builtin/push.c:694
 msgid "push options must not have new line characters"
-msgstr ""
+msgstr "опції push не повинні містити символів нового рядка"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "опції push не можуть містити символи нового рядка"
```

File: po/uk.po [hunk 5]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:42
 msgid "keep"
-msgstr ""
+msgstr "зберегти"
 
 #: builtin/reset.c:92
 msgid "You do not have a valid HEAD."
-msgstr ""
+msgstr "Ви не маєте припустимий HEAD."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "У вас немає дійсного HEAD."
```

File: po/uk.po [hunk 6]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
 
 #: builtin/reset.c:42
 msgid "mixed"
-msgstr ""
+msgstr "змішаний"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "змішане"
```

File: po/uk.po [hunk 7]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:42
 msgid "soft"
-msgstr ""
+msgstr "м’який"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "м’яке"
```

File: po/uk.po [hunk 8]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:42
 msgid "hard"
-msgstr ""
+msgstr "жорсткий"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "жорстке"
```

File: po/uk.po [hunk 9]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 #, c-format
 msgid "Failed to find tree of %s."
-msgstr ""
+msgstr "Не вдалося знайти дерево %s."
 
 #: builtin/reset.c:125
 #, c-format
 msgid "HEAD is now at %s"
-msgstr ""
+msgstr "HEAD зараз знаходиться на %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "HEAD зараз на %s"
```

File: po/uk.po [hunk 10]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:226
 #, c-format
 msgid "Cannot do a %s reset in the middle of a merge."
-msgstr ""
+msgstr "Неможливо виконати скидання %s посеред злиття."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Неможливо виконати %s скидання посеред злиття."
```

File: po/uk.po [hunk 11]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:324
 msgid "be quiet, only report errors"
-msgstr ""
+msgstr "мовчати, повідомляти лише про помилки"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "тихісенько, повідомляти лише про помилки"
```

File: po/uk.po [hunk 12]
```diff
@@ -1583,74 +1591,74 @@ msgstr ""
...
 
 #: builtin/reset.c:335
 msgid "reset HEAD but keep local changes"
-msgstr ""
+msgstr "скинути HEAD, але зберегти локальні зміни"
 
 #: builtin/reset.c:341
 msgid "record only the fact that removed paths will be added later"
-msgstr ""
+msgstr "записуйте лише той факт, що вилучені шляхи буде додано пізніше"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "записати лише той факт, що вилучені шляхи будуть додані пізніше"
```

**Review by arkid15r — COMMENTED** [review]
It's a decent piece of translation!
Getting closer!

File: po/uk.po (hunk 11 above)
Comment (kasya) [reply po/uk.po]: 😄🤫

**Pull request merged**